protocols/braintrust/overview.md

```diff
@@ -0,0 +1,62 @@
+# Braintrust
+
+## Overview
```

Comment:
Please put an empty line between subheaders and paragraphs.

Reply:
Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w